### ISSUE TITLE
Bug plotting PDOS

### ISSUE THREAD

**Issue by ajjackson** [SMTG-UCL/galore#21]
**Describe the bug**
Plotting PDOS is broken on master (12704888) with error message 
```  File "/home/adam/src/galore/galore/plot.py", line 132, in plot_pdos
    tdos = np.zeros(len(next(pdos_data.values())['energy']))
TypeError: 'ValuesView' object is not an iterator
```

**To Reproduce**
```galore vasprun.xml --pdos -p```

**Expected behavior**
Generate a plot and not an error message

**To fix**
- [x] **Fix the error**. This is pretty easy, just need to change the way this list is accessed. `tdos = np.zeros(len(next(pdos_data.items()[1])['energy']))` should work.
- [x] **Add a test** This problem should have been caught by unit testing. We need to check this code runs without raising errors, even if inspecting the plotted output is hard.

**Comment by ajjackson** [comment]
Seems to be fixed in #22 now, merge is imminent.

**Comment by ajjackson** [comment]
Merged and released
